frontend/express/public/javascripts/countly/vue/components/dropdown.js

```diff
@@ -187,6 +187,123 @@
         }
     }));
 
+    Vue.component("cly-vue-menubox", countlyBaseComponent.extend({
+        template: '<cly-dropdown ref="dropdown" v-on="$listeners" class="cly-vue-menubox-new__dropdown">\
```

Comment:
Class names should be updated in accordance with the component name change, of course.